app/design/adminhtml/default/openmage/template/forgotpassword.phtml

```diff
@@ -0,0 +1,73 @@
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title><?php echo Mage::helper('adminhtml')->__('Log into Magento Admin Page'); ?></title>
+    <link type="text/css" rel="stylesheet" href="<?php echo $this->getSkinUrl('login.css') ?>" media="all" />
+    <link rel="icon" href="<?php echo $this->getSkinUrl('favicon.ico'); ?>" type="image/x-icon" />
+    <link rel="shortcut icon" href="<?php echo $this->getSkinUrl('favicon.ico'); ?>" type="image/x-icon" />
+
+    <script type="text/javascript" src="<?php echo $this->getJsUrl(); ?>index.php/x.js?f=prototype/prototype.js,prototype/validation.js,mage/adminhtml/events.js,mage/adminhtml/form.js,scriptaculous/effects.js"></script>
+    <script type="text/javascript" src="<?php echo $this->getJsUrl('mage/captcha.js') ?>"></script>
+
+    <!--[if IE]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('iestyles.css'); ?>" type="text/css" media="all" /> <![endif]-->
+    <!--[if lt IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('below_ie7.css'); ?>" type="text/css" media="all" /> <![endif]-->
+    <!--[if IE 7]> <link rel="stylesheet" href="<?php echo $this->getSkinUrl('ie7.css'); ?>" type="text/css" media="all" /> <![endif]-->
+</head>
+<body id="page-login">
+    <div class="login-container">
+        <div class="login-box">
+            <form method="post" action="" id="loginForm">
+                <fieldset class="login-form">
+                    <input name="form_key" type="hidden" value="<?php echo $this->getFormKey(); ?>" />
+                    <h2><?php echo Mage::helper('adminhtml')->__('Forgot your user name or password?'); ?></h2>
+                    <div id="messages">
+                        <?php echo $this->getMessagesBlock()->toHtml(); ?>
+                    </div>
+                    <div class="input-box forgot-password"><label for="email"><?php echo Mage::helper('adminhtml')->__('Email Address:'); ?></label><br />
+                        <input type="text" id="email" name="email" value="" class="required-entry input-text forgot-password validate-email" style="width:461px;" />
+                    </div>
+                    <?php echo $this->getChildHtml('form.additional.info'); ?>
+                    <div class="clear"></div>
+                    <div class="form-buttons">
+                        <a class="left" href="<?php echo $this->getUrl('adminhtml', array('_nosecret' => true)); ?>">&laquo; <?php echo Mage::helper('adminhtml')->__('Back to Login'); ?></a>
+                        <input type="submit" class="form-button" onclick="loginForm.submit()" value="<?php echo Mage::helper('adminhtml')->__('Retrieve Password'); ?>">
+                    </div>
+                </fieldset>
+                <p class="legal"><?php echo Mage::helper('adminhtml')->__('OpenMage - Actively supported fork of Magento Community Edition') ?></p>
```

Comment:
Please just call it OpenMage LTS and do not mention Magento CE. See discussion in https://github.com/OpenMage/magento-lts/pull/1014